contracts/rewards/AutoCompoundingRewards.sol

```diff
@@ -366,7 +366,7 @@ contract AutoCompoundingRewards is IAutoCompoundingRewards, ReentrancyGuardUpgra
     /**
      * @dev processes the rewards of a given pool
      */
-    function _processRewards(Token pool, bool skip) private {
+    function _processRewards(Token pool, bool skipRecentlyProcessedPrograms) private {
```

Comment:
maybe `forceProcessing` and reverse the conditions?

Reply:
I agree with @barakman. It's something that should better require an opt-in (e.g., explicit `true`)

Reply:
sure, so maybe just a shorter name? `skipRecent`?

Reply:
done